deepchecks/utils/distribution/drift.py

```diff
@@ -177,18 +216,32 @@ def calc_drift_and_plot(train_column: pd.Series,
 
         dist_traces, dist_x_axis, dist_y_axis = feature_distribution_traces(train_dist, test_dist, value_name)
     elif column_type == 'categorical':
-        scorer_name = 'PSI'
-        expected, actual, _ = \
-            preprocess_2_cat_cols_to_same_bins(dist1=train_column, dist2=test_column,
-                                               max_num_categories=max_num_categories_for_drift)
-        expected_percents, actual_percents = expected / len(train_column), actual / len(test_column)
-        score = psi(expected_percents=expected_percents, actual_percents=actual_percents)
-
-        bar_traces, bar_x_axis, bar_y_axis = drift_score_bar_traces(score, bar_max=1)
-        dist_traces, dist_x_axis, dist_y_axis = feature_distribution_traces(
-            train_dist, test_dist, value_name, is_categorical=True, max_num_categories=max_num_categories_for_display,
-            show_categories_by=show_categories_by
-        )
+        if categorical_drift_method == 'Cramer':
+            scorer_name = 'Cramer\'s V'
+            score = cramers_v(dist1=train_dist, dist2=test_dist)
+
+            bar_traces, bar_x_axis, bar_y_axis = drift_score_bar_traces(score, bar_max=1)
+            dist_traces, dist_x_axis, dist_y_axis = feature_distribution_traces(
+                train_dist, test_dist, value_name, is_categorical=True,
+                max_num_categories=max_num_categories_for_display,
+            )
```

Comment:
the last 5 lines of code could be reused between both methods...

Reply:
Psi is a bit different, i can refactor the pso function tho